epochStart/bootstrap/process.go

```diff
@@ -1208,6 +1208,12 @@ func (e *epochStartBootstrap) Close() error {
 		log.LogIfError(err)
 	}
 
+	if !check.IfNil(e.dataPool) && !check.IfNil(e.dataPool.PeerAuthentications()) {
```

Comment:
If we close the peer authentication datapool here, it won't be able to clean the new entries after the bootstrap process is done and we will end up in OOM. I would leave this close to be called only when the node is shuffled out or it is closing.
This is done automatically by the runner instance (node package) that calls Close on all managed components that each calls Close on all contained sub-components.